spec/systems/users_spec.rb

```diff
@@ -0,0 +1,123 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :system do
+  let(:user) { create(:user) }
+
+  describe 'ログイン前' do
+    describe 'ユーザー新規登録' do
+      context 'フォームの入力値が正常' do
+        it 'ユーザーの新規作成が成功する' do
+          visit sign_up_path
+          fill_in 'Email', with:"sample@example.com"
+          fill_in 'Password', with:"foobar"
+          fill_in 'Password confirmation', with:"foobar"
+          click_button 'SignUp'
+          expect(page).to have_content 'User was successfully created.'
+          expect(current_path).to eq login_path
+        end
+      end
+
+      context 'メールアドレスが未入力' do
+        it 'ユーザーの新規作成が失敗する' do
+          visit sign_up_path
+          fill_in 'Email', with:" "
```

Comment:
空白文字にする必要がないので`''`か`nil`の方が分かりやすいと思います。
```suggestion
          fill_in 'Email', with: nil
```